app/reducers/index.js

```diff
@@ -7,6 +7,10 @@ const defaultState = {
       link: null,
       copied: false
     }
+  },
+  inspect: {
+    key: null,
```

Comment:
can this be simplified into one variable? just `inspect: { key: null }` or `inspect: { key: ... }`